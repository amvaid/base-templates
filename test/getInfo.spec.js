const getInfo = require('../packages/oc-template-jade/getInfo');

describe('getInfo method', () => {
  describe('when invoking the method', () => {
    const info = getInfo();

    test('should return the correct version', () => {
      expect(info.version).toBeDefined();
    });
    test('should return the correct template type', () => {
      expect(info.type).toBe('oc-template-jade');
    });
    test('should return the list of externals', () => {
      expect(info.externals).toMatchSnapshot();
    });
  });
});
