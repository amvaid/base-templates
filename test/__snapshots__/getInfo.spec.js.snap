// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getInfo method when invoking the method should return the list of externals 1`] = `
Array [
  Object {
    "global": "jade",
    "url": "https://unpkg.com/jade-legacy@1.11.1/runtime.js",
  },
]
`;
